spec/models/participant_profile_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe ParticipantProfile, type: :model do
+  before :each do
+    @object = create(:participant_profile)
+  end
+  let(:ect_profile) { create(:early_career_teacher_profile) }
+
+  describe "state transitions" do
+    it "should have an initial state of assigned" do
```

Comment:
It's the default preliminary state that a user has been added to the state transitions and is available to "Join" a course and become "active". "assigned" rather than "entered", since the user has already been entered via registration and exists, but the join event has not yet been triggered. That's almost certainly going to happen immediately from that state, but it doesn't seem good practice to initialise them in the "joined" state.